### PR TITLE
Calculator

### PR DESCRIPTION
**Pull request by killy10o10**
**Description:**
>In this feature :
>- I Created a directory called components.
>- Inside components, I created a new Calculator.js file.
>- In Calculator.js, I created a React component that matches the design wireframe from the curriculum.
⬇️⬇️⬇️Below is a screenshot of my final UI design.

calculator_screenshot

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
```
Comment (Deepakdanger) [inline src/App.js:3]: - [ ] **Requirement** The `App.js` component should be class-based.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @killy10o10, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (killy10o10) [reply src/App.js:3]: @Deepakdanger Thanks for the review. I don't fully understand what you mean by that. Can you kindly elaborate more?

Comment (Whoistolu) [reply src/App.js]: Well resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @killy10o10 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**